stable/redis-ha/values.yaml

```diff
@@ -66,9 +53,8 @@ persistentVolume:
   ##
   subPath: ""
 ## Redis image version
-redis_image: quay.io/smile/redis:4.0.2
+redis_image: quay.io/ianmaddox/redis-sentinel:4.0.6
```

Comment:
Can you add image source files to understand what you did ?

Reply:
Full changeset is here:
https://github.com/smileisak/docker-images/pull/4